backend/utils.py

```diff
@@ -92,7 +92,13 @@ def call_github_action(
         if params:
             payload["inputs"] = params
 
-    response = post(url, headers=headers, data=json.dumps(payload))
+    response: Response = post(url, headers=headers, data=json.dumps(payload))
+    if response.status_code >= 400:
+        raise RuntimeError(
+            f"Error calling GitHub action {action_name} with params:"
```

Comment:
## Now raises error
If error came back from GitHub API. I thought I was already doing this; guess not.